dbt_platform_helper/providers/platform_config_schema.py

```diff
@@ -518,18 +522,19 @@ def _valid_s3_bucket_arn(key):
             }
         )
 
-    @staticmethod
-    def __s3_bucket_schema() -> dict:
-        return PlatformConfigSchema.__valid_s3_base_definition() | {
-            "type": "s3",
-            Optional("objects"): [
-                {"key": str, Optional("body"): str, Optional("content_type"): str}
-            ],
-        }
-
     @staticmethod
     def __s3_bucket_policy_schema() -> dict:
-        return PlatformConfigSchema.__valid_s3_base_definition() | {"type": "s3-policy"}
```

Comment:
This was including a whole host of irrelevant schema options that were not used anywhere and were just causing confusion (in my head).